fuel/app/bootstrap.php

```diff
@@ -10,6 +10,8 @@
 // Register the autoloader
 \Autoloader::register();
 
+$hogehogheo = '123';
+$joiojijor = "ewjroiwerwerw";
```

Comment:
__[code_sniffer]__
* String "ewjroiwerwerw" does not require double quotes; use single quotes instead